api/src/app.ts

```diff
@@ -48,8 +50,14 @@ app.patch("/", function(req,res){
 
   //Deletes a Task
 app.delete("/delete/:taskId", async function(req,res){
-  await taskDataStore.deleteTask(req.params.taskId)
-  res.sendStatus(200);
+  console.log(req.params.taskId);
+  try{
+    await taskDataStore.deleteTask(req.params.taskId)
+    res.sendStatus(200);
+  }catch (err){
+    console.log(err);
```

Comment:
This should just send the error. Usually there would be some processing of the error itself to give better context to the user, but for the purposes of this project just returning the error to the client would be sufficient.